gilda/generate_terms.py

```diff
@@ -594,7 +595,12 @@ def main():
     terms = get_all_terms()
     from .resources import GROUNDING_TERMS_PATH as fname
     logger.info('Dumping into %s' % fname)
-    write_unicode_csv(fname, [t.to_list() for t in terms], delimiter='\t')
+    header = ['norm_text', 'text', 'db', 'id', 'entry_name', 'status',
+              'source', 'organism']
+    with gzip.open(fname, 'wt', encoding='utf-8') as fh:
+        writer = csv.writer(fh, delimiter='\t')
+        writer.writerow(header)
+        writer.writerows([t.to_list() for t in terms])
```

Comment:
not like it makes a difference but you could do 

```suggestion
        writer.writerows(t.to_list() for t in terms)
```

So this is a generator instead of rasterizing the list. If this gets much bigger, somehow this is better, especially if the `get_all_terms()` can be turned into an iterator too - it will save memory